nodes.py

```diff
@@ -174,16 +179,100 @@ def run(self, image: Tensor, multiple_of: int, method: str) -> Tuple[Tensor]:
             return (image[:, top:bottom, left:right, :],)
 
 
+class HFHubLoraLoader:
+    def __init__(self):
+        self.loaded_lora = None
```

Comment:
Dont make this an implicit tuple,
please destruct it into self.loaded_lora_path and self.loaded_lora